testserv.c

```diff
@@ -363,6 +420,30 @@ cttest_peek_not_found()
     ckresp(fd, "NOT_FOUND\r\n");
 }
 
+void
+cttest_peek_ok_unix()
+{
+    char *name = SERVER_UNIX();
+    int fd = mustdialunix(name);
+    mustsend(fd, "put 0 0 1 1\r\n");
+    mustsend(fd, "a\r\n");
+    ckresp(fd, "INSERTED 1\r\n");
+
+    mustsend(fd, "peek 1\r\n");
+    ckresp(fd, "FOUND 1 1\r\n");
+    ckresp(fd, "a\r\n");
+
+    unlink(name);
+}
+
+void
+cttest_unix_auto_removal()
+{
+    // Twice, to trigger autoremoval
+    SERVER_UNIX();
+    SERVER_UNIX();
```

Comment:
Do we expect that previous server instance should be still running here? If not we could kill it with kill_srvpid();

Reply:
Not really.  So yes, killing the server in between makes sense.